src/manifest.c

```diff
@@ -727,6 +742,97 @@ static int write_manifest_signature(struct manifest *manifest, const char *suffi
 }
 
 /* Returns 0 == success, -1 == failure */
+static int write_includes_plain(struct manifest *manifest)
+{
+	FILE *out = NULL;
+	GList *submanifests;
+	GHashTable *manifest_includes = g_hash_table_new(g_str_hash,
+							 g_str_equal);
+	char *base = NULL;
+	char *conf = config_output_dir();
+	char *dir = NULL;
+	char *filename = NULL;
+	int ret = -1;
+	unsigned int count = 0;
+
+	/* ignore non MoM manifests */
+	if (strcmp(manifest->component, "MoM")) {
+                ret = 0;
+		goto exit;
+	}
+
+	if (conf == NULL) {
+		assert(0);
+	}
+	string_or_die(&filename, "%s/%i/Manifest.%s.includes", conf,
+		      manifest->version, manifest->component);
```

Comment:
Because the file body of this new file is completely different than the other manifests, should we call it a manifest? I would be okay calling it an "includes-list" or similar.